syntax = "proto3";

package syft.core.node.common.action;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message RunClassMethodAction {
  string path = 1;
  bytes _self = 2;
  repeated bytes args = 3;
  map<string, bytes> kwargs = 4;
  syft.core.common.UID id_at_location = 5;
  syft.core.io.Address address = 6;
  syft.core.common.UID msg_id = 7;
}
